tox.ini

```diff
@@ -21,6 +21,7 @@ passenv =
     USER
     WORKSPACE
     WORKSPACE_TMP
+    OIDC_CA
```

Comment:
Instead of passing `CURL_CA_BUNDLE` or `OIDC_CA` in, perhaps we should _set_ it here at line 26 (and, therefore, perhaps we can use `CURL_CA_BUNDLE` rather than a custom one).